prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -62,26 +70,55 @@ class LoginCommand : OktaCommand(
         .choice("local", "test", "staging", "prod")
         .default("local", "local")
 
+    private val useApiKey by option(
+        "--useApiKey",
+        help = "Enable to sign in to Okta via an API key instead of a login page"
```

Comment:
This should point to a place where the user can learn how to set up their env vars. See main comment.